src/helpers/strings.ts

```diff
@@ -130,3 +179,112 @@ export function formatCurrency(
 
   return `${currencySymbol}${formattedWhole}.${formattedFraction}`;
 }
+
+export function formatFractionWorklet(fraction: string): string {
```

Comment:
It's safe to call worklet functions from non worklet contexts right? If so we shouldn't need to have two versions of this function, just make the main one a worklet